docs/current/cloud/572923-get-started.md

```diff
@@ -0,0 +1,313 @@
+
+1. Update your GitLab CI workflow and add the variable to your CI environment. The environment variable must be named `DAGGER_CLOUD_TOKEN`. Refer to the GitLab documentation on [using CI/CD variables](https://docs.gitlab.com/ee/ci/variables/index.html#use-cicd-variables-in-job-scripts).
+
+1. GitLab makes some metadata available only to merge request pipelines. If your CI workflow relies entirely on merge requests, you can improve your Dagger Cloud experience by optionally adding a workflow rule to create a pipeline only on merge requests. Refer to the GitLab documentation on [merge requests](https://docs.gitlab.com/ee/user/project/merge_requests/) and [workflow rules](https://docs.gitlab.com/ee/ci/yaml/workflow.html#switch-between-branch-pipelines-and-merge-request-pipelines).
+
+  :::danger
+  This step is optional and should be performed only after an in-depth analysis of the impact on your CI workflow.
+  :::
+
+Here is a sample GitLab CI workflow with the Dagger Cloud integration highlighted:
+
+```yaml title=".gitlab-ci.yml" file=./snippets/get-started/ci/gitlab.yml
+```
+
+</TabItem>
+<TabItem value="CircleCI">
+
+1. Create a new environment variable in your CircleCI project named `DAGGER_CLOUD_TOKEN` and set it to the value of the token obtained in [Step 1](#step-1-sign-up-for-dagger-cloud). Refer to the CircleCI documentation on [creating environment variables for a project](https://circleci.com/docs/set-environment-variable/#set-an-environment-variable-in-a-project).
+
+1. If your source code is stored in a GitHub, GitLab or Atlassian Bitbucket repository, update your CircleCI workflow and add the following pipeline values to the CI environment. Refer to the CircleCI documentation on [using pipeline values](https://circleci.com/docs/variables/#pipeline-values).
+
+  GitHub:
+
+  ```yaml
+  environment:
+    CIRCLE_PIPELINE_NUMBER: << pipeline.number
+  ```
+
+  GitLab:
+
+  ```yaml
+  environment:
+    CIRCLE_PIPELINE_NUMBER: << pipeline.number >>
+    CIRCLE_PIPELINE_TRIGGER_LOGIN: << pipeline.trigger_parameters.gitlab.user_username >>
+    CIRCLE_PIPELINE_REPO_URL: << pipeline.trigger_parameters.gitlab.repo_url >>
+    CIRCLE_PIPELINE_REPO_FULL_NAME: << pipeline.trigger_parameters.gitlab.repo_name >>
+  ```
+
+  Atlassian BitBucket:
+
+  ```yaml
+  environment:
+    CIRCLE_PIPELINE_NUMBER: << pipeline.number
+  ```
+
+Here is a sample CircleCI workflow. The `DAGGER_CLOUD_TOKEN` environment variable will be automatically injected.
+
+```yaml title=".circleci/config.yml" file=./snippets/get-started/ci/circle.yml
```

Comment:
This requires a re-test, as it was not tested after the special Circle variables were added.

Reply:
@kpenfound would you be able to test this?